ui/lib/pki/addon/components/key-usage.hbs

```diff
@@ -9,58 +9,24 @@
   />
   {{#if (get @model prop)}}
     <div class="box is-tall is-marginless" data-test-surrounding-div={{@group}}>
-      <FormFieldLabel
-        for="keyUsageLabel"
+      <CheckboxList
```

Comment:
👏 ❤️ 😍 I was just wondering how to make this component reusable after seeing Ivana's designs yesterday and seeing layout make more of an appearance. Well done!